content/en/docs/release-oversight/testing-in-production.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Testing in \"production\""
+description: Running our development tests on our latest stable sprint release.
+---
+
+### Background
+Like security, quality comes in layers.  In addition to the directly signal from our CI, [release gate testing](/docs/architecture/release-gating/) and Quality Engineering, we're now testing our sprint output on a critical piece of our own infrastructure.
+
+### Our philosophy on failure
+It's important to understand we believe breaking ourselves first is a form of success.  Failing a sprint is a gift in this sense.  It gives us time to adjust before our next big release.  What this means in practice is that, when sprints fail, we commit to clearing our sprint of feature work and focusing on stability across whatever teams necessary.
+
+### Sprint success or failure
+There are couple ways we judge the success or a failure of a sprint:
+
+* Did we have a build on time?
+* Were any changes required by the administrator not covered by an [admin ack](https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare)?
+* Did we find a significant problem that forced us to remove the production cluster from rotation?
+
+### Selecting the build
+The selection process is quite simple.  On the last Wednesday of each sprint the [Technical Release Team](/docs/release-oversight/the-technical-release-team/) (TRT) will select an accepted payload from the last two days.  From there Quality Engineering will run additional tests.  Barring any new surprises Test Platform will be notified that a release is ready.
+
+It's worth noting that we'll likely create a dedicated channel for managing these releases through the [Update Service](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html) in the upcoming months.
+
+### The environment
+We've selected one of our build farm clusters to run the output of each sprint.  This is a critical piece of infrastructure that all development teams are motivated to keep working.  In this document we'll refer to this cluster as the *sacrificial cluster*.
+
+### Investigating problems
+The TRT has admin access to the sacrificial cluster for debugging purposes.  As needed this access can be temporarily granted by Test Platform to any other development team that needs it.  Temporary configuration changes are allowed for experiments but anything needed to persist will require failing the sprint.
```

Comment:
> As needed this access can be temporarily granted...

Ideally this sentence links to process docs, so that in the excitement of "oh no, we need to debug this thing!" we have a procedure written in the calm before-times that we can follow, instead of trying to build the process on the fly.